babel/src/test/resources/sql/big-query.iq

```diff
@@ -371,6 +371,76 @@ SELECT EXTRACT(DAY FROM DATE '2013-12-25') AS the_day;
 
 !ok
 
+SELECT EXTRACT(MONTH FROM INTERVAL -1 MONTHS) as extr;
```

Comment:
make sure your first commit message(the CR title) is begin with [`${jira_ID}`]

Reply:
@zoudan thanks, all done i hope.